servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -46,6 +47,10 @@
     private static final String KEY_PREV_TIME = "prevTime";
     private static final String KEY_PREV_IP = "prevIP";
     private static final String KEY_CLIENT_CERT = "clientCert";
+    private static final String KEY_LAST_NOTIFIED_TIME = "lastNotifiedTime";
```

Comment:
Added 4 new nullable columns. Added tests that verify get / insert / update works correctly with null values for these columns.